docs/app/layout.tsx

```diff
@@ -0,0 +1,19 @@
+import "./global.css";
+import { RootProvider } from "fumadocs-ui/provider";
+import { Inter } from "next/font/google";
+import type { ReactNode } from "react";
+import { GoogleAnalytics } from "@next/third-parties/google";
+const inter = Inter({
+  subsets: ["latin"],
+});
+
+export default function Layout({ children }: { children: ReactNode }) {
+  return (
+    <html lang="en" className={inter.className} suppressHydrationWarning>
+      <body className="flex flex-col min-h-screen">
+        <RootProvider>{children}</RootProvider>
+      </body>
+      <GoogleAnalytics gaId="" />
```

Comment:
This doesn't seem to be used currently. Do you have plans to suply a proper `gaId`? If not, I think we should remove it. Similarly, we can also remove the `@next/third-parties/google` package if it's only being used here.

Reply:
Yes, was going to ask if you would prefer to create one, or are okay with using one created by Foundation.
In general, I would like to get more visibility into the traffic for these docs to know which pages to prioritize.

Reply:
@acheroncrypto added google analytics id